Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Prefabs/Logout.cs

```diff
@@ -26,4 +20,11 @@ public async void OnLogout()
         // Go back to the first scene to log in again
         SceneManager.LoadScene(0);
     }
+
+    private void OnApplicationQuit()
+    {
+        Debug.Log("Disconnecting wallet...");
+        
+        OnLogout();
```

Comment:
This one would cause `SceneManager.LoadScene(0);`. Not sure what this would result in, but this might give us unintended performance spikes. Move out everything except for scene loading in %HandleLogout% method perhaps?

Reply:
Also if you didn't login this will throw when trying to get Web3 in `await Web3Accessor.Web3.TerminateAsync()`

Reply:
> This one would cause `SceneManager.LoadScene(0);`. Not sure what this would result in, but this might give us unintended performance spikes. Move out everything except for scene loading in %HandleLogout% method perhaps?

will do

Reply:
> Also if you didn't login this will throw when trying to get Web3 in `await Web3Accessor.Web3.TerminateAsync()`

this shouldn't trigger unless we're logged in, since it's a Monobehaviour on Logout button

Reply:
> > This one would cause `SceneManager.LoadScene(0);`. Not sure what this would result in, but this might give us unintended performance spikes. Move out everything except for scene loading in %HandleLogout% method perhaps?
> 
> will do

done